documentation/API-design-guidelines.md

```diff
@@ -1287,44 +1288,51 @@ response:
+CAMARA event notification leverages **CloudEvents** as it is a vendor-neutral specification for defining the format of event data. A generic neutral CloudEvent notification swagger is available in commonalities/artifact directory (notification-as-cloud-event.yaml).
+
+Note: The notification is the message posted on listener side. We describe the notification(s) in the CAMARA API using the `callbacks`. From API consumer it could confusing because this endpoint must be implemented on the business API consumer side. This notice should be explicited mentioned in all CAMARA API documentation featuring notifications.
 
-Only Operation POST is provided for `eventNotifications` and the expected response code is `204`.
+Only Operation POST is provided for `eventNotifications` and the expected response code is `204`. 
+The url for this `POST` operation must be specified in the swagger as `{$request.body#/webhook/notificationUrl}`. 
 
-For consistence among CAMARA APIs the uniform `eventNotifications` model must be used:
+For consistence among CAMARA APIs the uniform CloudEvents model must be used with following rules:
 
 | name | type | attribute description | cardinality |
 | ----- |	-----  |	 -----  |  -----  | 
-| eventSubscriptionId | string | subscription identifier - could be valued for Resource-based subscription | optional |
-| event | object | event structure - see next table | mandatory |
+| id | string | identifier of this event, that must be unique in the source context. | mandatory |
+| source | string - URI | identifies the context in which an event happened in the specific Provider Implementation. Often this will include information such as the type of the event source, the organization publishing the event or the process that produced the event. The exact syntax and semantics behind the data encoded in the URI is defined by the event producer. | mandatory |
+| type | string | a value describing the type of event related to the originating occurrence. For consistency accross API we mandate following pattern: org.camara.api._name of the API_._event name_ (for exemple org.camara.api.device-status.RoamingStatusEvent ) | mandatory |
+| specversion | string | version of the specification to which this event conforms (must be 1.0 if it conforms to cloudevents 1.0.2 version) | mandatory |
+| datacontenttype | string | media-type that describes the event payload encoding, must be `application/json for CAMARA APIs` | optional |
+| subject | string | describes the subject of the event. In CAMARA we enforce to use subject as defined in each API. This attribute is tagged as optional in CloudEvents specification but from CAMARA perspective we **strongly** recommend to value this attribute. | optional |
+| time | string  datetime| timestamp of when the occurrence happened (must adhere on CAMARA datetime recommendation based on RFC 3339) | optional |
```

Comment:
Think that in CAMARA we have to consider "time" mandatory, as we are using this replacing "former" model within SusbcriptionDetail

Reply:
Do you think we can be more restrictive than CloudEvnts ? 
We can add in the description "_This attribute is tagged as optional in CloudEvents specification but from CAMARA perspective we **strongly** recommend to value this attribute_."

Reply:
I think we can define "CAMARA profile" of CloudEvents - the change from _optional_ to _mandatory_ or _recommended_ to _required_ does not complicate development of API implementations or applications, it can even simplify the task.

Reply:
Get your points Ludovic, Rafał. 

I think at least we can add a sentence in that way, just to reflect the relevance of this info. 
If quorom, also imho could be mandatory (not a real problem being optional). Some rationale below:

In TEF:
- From technical perspective we can implement it
- From Business perspective, we think is really valuable information as it indicates when the event occurs. There can be problems in systems, retries,...and this information guarantees API consumer when the event really happened, because it may not consider that the reception timestamp is the event timestamp and in some business cases such information can be the "key" one (e.g. fraud prevention, enter into roaming,...)

Reply:
updated accordingly to you comments (switched both to mandatory)